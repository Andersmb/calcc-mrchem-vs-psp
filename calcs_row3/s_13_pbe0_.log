Starting job 5688137 on c1-[21,26],c2-39,c3-[13,27],c5-22 at Fri May 20 08:34:19 CEST 2022

Copying s_13_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying s_13_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688137      s_13_pbe0_        180                                             01:29:46      0:0 
5688137.bat+      batch         24        1   00:00:04          0   00:00:04   01:29:46      0:0 
5688137.ext+     extern        180        6   00:00:00          2   00:00:00   01:29:46      0:0 
5688137.0      mrchem.x        180       30   01:29:18         29   03:40:17   01:29:27      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688137                                                                          
5688137.bat+     19328K          0     19328K        0              0          0 
5688137.ext+          0          2          0        0              2          0 
5688137.0     17173236K         29 6634630075        0             11          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688137                                                                                               
5688137.bat+       16.90M               0         16.90M        0.03M                0          0.03M 
5688137.ext+        0.00M               2          0.00M            0                2              0 
5688137.0           6.75M               2          6.73M      133.65M               22         75.21M 

Job 5688137 completed at Fri May 20 10:04:05 CEST 2022
