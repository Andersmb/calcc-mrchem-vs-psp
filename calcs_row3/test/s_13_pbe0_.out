                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       O            :        0.000000       -0.668487       -3.473260
    1       S            :        0.000000       -0.668487       -0.704812
    2       S            :        0.000000        1.002030        2.437803
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              20
 Beta electrons          :                                              20
 Total electrons         :                                              40
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999995935295e-01
    1    1    a          :                              1.000000393773e+00
    2    1    a          :                              1.000000019623e+00
    3    1    a          :                              1.000000001884e+00
    4    1    a          :                              9.999999961583e-01
    5    1    a          :                              9.999999991466e-01
    6    1    a          :                              9.999999968690e-01
    7    1    a          :                              9.999998064043e-01
    8    1    a          :                              9.999999998356e-01
    9    1    a          :                              1.000000218100e+00
   10    1    a          :                              1.000000088248e+00
   11    1    a          :                              9.999999996617e-01
   12    1    a          :                              9.999999984895e-01
   13    1    a          :                              9.999999987674e-01
   14    1    a          :                              9.999999923520e-01
   15    1    a          :                              9.999999697608e-01
   16    1    a          :                              9.999999936574e-01
   17    1    a          :                              1.000000049863e+00
   18    1    a          :                              1.000000009195e+00
   19    1    a          :                              9.999999890465e-01
   20    1    b          :                              9.999999528484e-01
   21    1    b          :                              9.999999930748e-01
   22    1    b          :                              9.999999827338e-01
   23    1    b          :                              1.000000000383e+00
   24    1    b          :                              9.999999964067e-01
   25    1    b          :                              9.999999992559e-01
   26    1    b          :                              9.999999999497e-01
   27    1    b          :                              9.999999968329e-01
   28    1    b          :                              1.000000006384e+00
   29    1    b          :                              9.999999516933e-01
   30    1    b          :                              9.999999796149e-01
   31    1    b          :                              9.999999996650e-01
   32    1    b          :                              9.999999984678e-01
   33    1    b          :                              1.000000473009e+00
   34    1    b          :                              9.999999938155e-01
   35    1    b          :                              9.999999685509e-01
   36    1    b          :                              1.000000004543e+00
   37    1    b          :                              1.000000050085e+00
   38    1    b          :                              1.000000009206e+00
   39    1    b          :                              9.999999902167e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                864.878708740231
 E-N energy              :            (au)              -2337.365410807814
 Coulomb energy          :            (au)                526.558330985402
 Exchange energy         :            (au)                -14.564492682234
 X-C energy              :            (au)                -45.056020161460
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                139.003255007381
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)              -1005.548883925874
 Nuclear energy          :            (au)                139.003255007381
---------------------------------------------------------------------------
 Total energy            :            (au)             -8.665456289185e+02
                         :      (kcal/mol)             -5.437655874669e+05
                         :        (kJ/mol)             -2.275115219438e+06
                         :            (eV)             -2.357990677190e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -88.826496719477
    1    1    a          :            (au)                -88.358549922752
    2    1    a          :            (au)                -18.851688895976
    3    1    a          :            (au)                 -8.320681349072
    4    1    a          :            (au)                 -7.919470017586
    5    1    a          :            (au)                 -6.266437112037
    6    1    a          :            (au)                 -6.264300772094
    7    1    a          :            (au)                 -6.255043535897
    8    1    a          :            (au)                 -5.864725456295
    9    1    a          :            (au)                 -5.858301189053
   10    1    a          :            (au)                 -5.851235512390
   11    1    a          :            (au)                 -1.126994707697
   12    1    a          :            (au)                 -0.929799114495
   13    1    a          :            (au)                 -0.594419191262
   14    1    a          :            (au)                 -0.484165639271
   15    1    a          :            (au)                 -0.480539364535
   16    1    a          :            (au)                 -0.388543050905
   17    1    a          :            (au)                 -0.269968274320
   18    1    a          :            (au)                 -0.257603531682
   19    1    a          :            (au)                 -0.189070611199
   20    1    b          :            (au)                -88.826496352388
   21    1    b          :            (au)                -88.358542828654
   22    1    b          :            (au)                -18.851677176250
   23    1    b          :            (au)                 -8.320664026872
   24    1    b          :            (au)                 -7.919465536872
   25    1    b          :            (au)                 -6.266444152110
   26    1    b          :            (au)                 -6.264304213365
   27    1    b          :            (au)                 -6.255058029294
   28    1    b          :            (au)                 -5.864734301095
   29    1    b          :            (au)                 -5.858304593653
   30    1    b          :            (au)                 -5.851240643580
   31    1    b          :            (au)                 -1.126988729249
   32    1    b          :            (au)                 -0.929808349307
   33    1    b          :            (au)                 -0.594416127121
   34    1    b          :            (au)                 -0.484166146691
   35    1    b          :            (au)                 -0.480537809271
   36    1    b          :            (au)                 -0.388541660775
   37    1    b          :            (au)                 -0.269968073975
   38    1    b          :            (au)                 -0.257601077576
   39    1    b          :            (au)                 -0.189065259322
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -506.716059055415
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          6.324555e+00        -866.545628918494         -8.665456e+02
    1          4.029719e-01        -871.040661817073         -4.495033e+00
    2          1.624640e-01        -871.212332253730         -1.716704e-01
    3          8.816071e-02        -871.246529139139         -3.419689e-02
    4          5.709960e-02        -871.258418584588         -1.188945e-02
    5          8.754084e-02        -871.269586586238         -1.116800e-02
    6          2.531773e-02        -871.271318224053         -1.731638e-03
    7          3.808121e-02        -871.272787211951         -1.468988e-03
    8          2.681696e-02        -871.273226696149         -4.394842e-04
    9          4.248524e-02        -871.273806204539         -5.795084e-04
   10          2.339399e-02        -871.274225523997         -4.193195e-04
   11          1.914481e-02        -871.274432517971         -2.069940e-04
   12          1.188375e-02        -871.274515928299         -8.341033e-05
   13          4.900504e-03        -871.274547604300         -3.167600e-05
   14          3.162002e-03        -871.274565374612         -1.777031e-05
   15          1.730082e-03        -871.274570168633         -4.794021e-06
   16          9.683696e-04        -871.274571582891         -1.414258e-06
   17          1.120837e-03        -871.274572889131         -1.306240e-06
   18          9.525972e-04        -871.274573508797         -6.196666e-07
   19          8.030336e-04        -871.274573895753         -3.869554e-07
   20          9.058223e-04        -871.274574262411         -3.666579e-07
   21          7.058136e-04        -871.274574525832         -2.634208e-07
   22          6.484650e-04        -871.274574692683         -1.668514e-07
   23          5.573775e-04        -871.274574810492         -1.178096e-07
   24          3.576759e-04        -871.274574896678         -8.618588e-08
   25          2.523325e-04        -871.274574939333         -4.265507e-08
   26          1.944239e-04        -871.274574963404         -2.407046e-08
   27          2.140052e-04        -871.274574993602         -3.019852e-08
   28          1.856068e-04        -871.274575014417         -2.081447e-08
   29          9.984441e-05        -871.274575021613         -7.195922e-09
---------------------------------------------------------------------------
                      SCF converged in 29 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       O            :        0.000000       -0.668487       -3.473260
    1       S            :        0.000000       -0.668487       -0.704812
    2       S            :        0.000000        1.002030        2.437803
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                869.975484042431
 E-N energy              :            (au)              -2348.894266609132
 Coulomb energy          :            (au)                528.217841935002
 Exchange energy         :            (au)                -14.542863495064
 X-C energy              :            (au)                -45.034025902231
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                139.003255007381
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)              -1010.277830028993
 Nuclear energy          :            (au)                139.003255007381
---------------------------------------------------------------------------
 Total energy            :            (au)             -8.712745750216e+02
                         :      (kcal/mol)             -5.467330459250e+05
                         :        (kJ/mol)             -2.287531065635e+06
                         :            (eV)             -2.370858794519e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -89.172792478244
    1    1    a          :            (au)                -89.072635844155
    2    1    a          :            (au)                -19.271476568992
    3    1    a          :            (au)                 -8.136211615628
    4    1    a          :            (au)                 -8.053370743384
    5    1    a          :            (au)                 -6.071810711299
    6    1    a          :            (au)                 -6.070790371795
    7    1    a          :            (au)                 -6.067497456769
    8    1    a          :            (au)                 -5.988690338695
    9    1    a          :            (au)                 -5.987478003515
   10    1    a          :            (au)                 -5.981789571555
   11    1    a          :            (au)                 -1.120367842072
   12    1    a          :            (au)                 -0.857567489674
   13    1    a          :            (au)                 -0.628040980333
   14    1    a          :            (au)                 -0.494220411055
   15    1    a          :            (au)                 -0.471941125639
   16    1    a          :            (au)                 -0.469003159526
   17    1    a          :            (au)                 -0.346609623515
   18    1    a          :            (au)                 -0.345585661717
   19    1    a          :            (au)                 -0.267740842254
   20    1    b          :            (au)                -89.172792478417
   21    1    b          :            (au)                -89.072635841423
   22    1    b          :            (au)                -19.271476577443
   23    1    b          :            (au)                 -8.136211616393
   24    1    b          :            (au)                 -8.053370740332
   25    1    b          :            (au)                 -6.071810714474
   26    1    b          :            (au)                 -6.070790372894
   27    1    b          :            (au)                 -6.067497456244
   28    1    b          :            (au)                 -5.988690337073
   29    1    b          :            (au)                 -5.987477994881
   30    1    b          :            (au)                 -5.981789567868
   31    1    b          :            (au)                 -1.120367860085
   32    1    b          :            (au)                 -0.857567493764
   33    1    b          :            (au)                 -0.628040974026
   34    1    b          :            (au)                 -0.494220414522
   35    1    b          :            (au)                 -0.471941154078
   36    1    b          :            (au)                 -0.469003158236
   37    1    b          :            (au)                 -0.346609616814
   38    1    b          :            (au)                 -0.345585647605
   39    1    b          :            (au)                 -0.267740842872
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -509.751241699260
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000022        0.092180        0.629982
 Magnitude               :            (au)                        0.636690
                         :         (Debye)                        1.618305
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000       -0.011207       -0.058231
 Magnitude               :            (au)                        0.059299
                         :         (Debye)                        0.150724
---------------------------------------------------------------------------
 Total vector            :        0.000022        0.080973        0.571751
 Magnitude               :            (au)                        0.577457
                         :         (Debye)                        1.467748
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  1h 29m 22s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
